exporter/clients.go

```diff
@@ -11,12 +11,16 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type ClientInfo struct {
+	Name, User, Created_at, Idle_since, Flags, Db, Omem, Cmd, Host, Port string
```

Comment:
Let's put these on separate lines and use CamelCase (e.g. IdleSince, not Idle_since)
Omem should be OMem